### DISCUSSION TITLE
4.8.4 release scheduled anytime soon?

### DISCUSSION THREAD

**Discussion by cnicolaescu** [ccache/ccache#1367] [General]
Hello,

I would like to know if there is a 4.8.4 release scheduled anytime soon?
I would like to test the changes (added since 4.8.3) in regards to Windows paths handling, as I'm facing cache misses on Windows that appear to be fixed already on `master`.

Thanks!

**Comment by jrosdahl** [comment]
Hi there! In general there is no plan or schedule for ccache releases – see #1137. That said, it's not unlikely that there will be a release in December.
